<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Coordinate</class>
 <widget class="QDialog" name="Coordinate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>54</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QComboBox" name="cbox_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string extracomment="Coordinate"/>
        </property>
        <property name="icon">
         <iconset resource="../../src.qrc">
          <normaloff>:/icon/resource/icon/coordinate.svg</normaloff>:/icon/resource/icon/coordinate.svg</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="PointSize"/>
        </property>
        <property name="icon">
         <iconset resource="../../src.qrc">
          <normaloff>:/icon/resource/icon/drag-surface.svg</normaloff>:/icon/resource/icon/drag-surface.svg</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspin_coord">
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/add.svg</normaloff>:/icon/resource/icon/add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/apply1.svg</normaloff>:/icon/resource/icon/apply1.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/reset.svg</normaloff>:/icon/resource/icon/reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/close-copy.svg</normaloff>:/icon/resource/icon/close-copy.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
